PullToRefresh/PullToRefresh.swift

```diff
@@ -47,7 +47,7 @@ public class PullToRefresh: NSObject {
     
     // MARK: - State
     
-    var state: State = .Initial {
+    public private(set) var state: State = .Initial {
```

Comment:
No particular reason. It might be excessive, however when it still compiled (so it wasn't used outside this class), and it wasn't set to public I assumed it should be considered private.